bytes-extended/src/main.sw

```diff
@@ -115,12 +144,12 @@ impl Bytes {
 
     /// Writes a b256 at the specified offset. Reverts if it violates the
     /// bounds of self.
-    pub fn write_b256(ref mut self, offset: u64, value: b256) {
+    pub fn write_b256(ref mut self, offset: u64, value: b256) -> u64 {
```

Comment:
could we document what the u64 returned by all these write_xxx fns is?

Also would be good to add the returned offset to the test coverage